.github/pull_request_template.md

```diff
@@ -1,20 +1,22 @@
 <!--
 If you are a first-time contributor to MontePy,
-refer the developing guidelines at:
+refer to the developing guidelines at:
 https://idaholab.github.io/MontePy/developing.html
 -->
 
 # Description
 
-Please include a summary of the change and which issue is fixed if applicable. Please also include relevant motivation and context.
+Please include a summary of the change and which issue is fixed, if applicable. Please also include relevant motivation and context.
 
 Fixes # (issue)
 
 # Checklist
 
-- [ ] I have performed a self-review of my own code
-- [ ] I have made corresponding changes to the documentation (if applicable)
-- [ ] I have added tests that prove my fix is effective or that my feature works (if applicable)
+- [ ] I have performed a self-review of my own code.
+- [ ] The code follows the standards outlined in the [development documentation](https://idaholab.github.io/MontePy/developing.html).
```

Comment:
I don't think this alone adds much value, as the guidelines are already linked. 

I would say remove this line and then go through the checklist here, and make sure everything is coverered: https://www.montepy.org/developing.html#merge-checklist